src/pyhf/schema/validator.py

```diff
@@ -0,0 +1,41 @@
+import jsonschema
+import pyhf.exceptions
+from pyhf.schema.loader import load_schema
+from pyhf.schema import variables
+from typing import Union
+
+
+def validate(spec: dict, schema_name: str, version: Union[str, None] = None):
```

Comment:
As we don't have `mypy` in the tests yet have these been verified yet?

Reply:
yes. i ran locally. `Union[str, None]` can become `Optional[str]` but I was going to pick that up in a different set of sweeping changes.